docs/diagnostic-logs.md

```diff
@@ -0,0 +1,197 @@
+
+Diagnostic logs are disabled by default. To enable diagnostic logs, follow these steps:
+
+1.	In the [Azure portal](https://portal.azure.com), under **Monitoring**, click **Diagnostic settings**.
+
+    ![Pane navigation to diagnostic settings](./images/diagnostic-logs/diagnostic-settings-menu-item.png)
+
+1.	Then click **Add diagnostic setting**.
+
+	![Add diagnostic logs](./images/diagnostic-logs/add-diagnostic-setting.png)
+
+1.	Set the archive target that you want. Currently, we support **Archive to a storage account** and **Send to Log Analytics**.
+
+1. Select the logs you want to archive.
+
+	![Diagnostics settings pane](./images/diagnostic-logs/diagnostics-settings-pane.png)
+
+
+1.	Save the new diagnostics settings.
+
+New settings take effect in about 10 minutes. After that, logs appear in the configured archival target, in the **Diagnostics logs** pane.
+
+For more information about configuring diagnostics, see the [overview of Azure diagnostic logs](../azure-monitor/platform/resource-logs-overview.md).
+
+### Diagnostic logs categories
+
+Azure SignalR Service captures diagnostic logs in one category:
+
+* **All Logs**: Track connections that connect to Azure SignalR Service. The logs Provide infomation about the connect/disconnect, authentication and throttling. For more information, see the next section.
+
+### Archive to a storage account
+
+Logs are stored in the storage account that configured in **Diagnostics logs** pane. A container named `insights-logs-alllogs` is created automatically to store diagnostic logs. Inside the container, logs are stored in the file `resourceId=/SUBSCRIPTIONS/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/RESOURCEGROUPS/XXXX/PROVIDERS/MICROSOFT.SIGNALRSERVICE/SIGNALR/XXX/y=YYYY/m=MM/d=DD/h=HH/m=00/PT1H.json`. Basically, the path is combined by `resource ID` and `Date Time`. The log files are splitted by `hour`. Therefore, the minutes always be `m=00`.
+
+All logs are stored in JavaScript Object Notation (JSON) format. Each entry has string fields that use the format described in the following sections.
+
+Archive log JSON strings include elements listed in the following tables:
+
+Name | Description
+------- | -------
+time | Log event time
+level | Log event level
+resourceId | Resource ID of your Azure SignalR Service
+location | Location of your Azure SignalR Service
+category | Catagory of the log event
+operationName | Operation name of the event
+callerIpAddress | IP address of your server/client
+properties | Detailed properties related to this log event. For more detail, see [`Properties Table`](#properties-table)
+
+<a href="properties-table"></a>
+**Properties Table**
+
+Name | Description
+------- | -------
+type | Collection of the log event. Currently, we provide information about connectivity to the Azure SignalR Service. Only `ConnectivityLogs` collection is available
+collection | Type of the log event. Allowed values are: `Connection`, `Authorization` and `Throttling`
```

Comment:
The descriptions seem to be wrong way around? (Why type's description is about collection and collection's description is about type?)

Reply:
No. the type is about different log types, collection is different set in the same log type

Reply:
But this looks very confusing. Why type is collection of the log event and collection is type of the log event.

Reply:
Oh, I see...